packages/atlas/src/views/viewer/MemberView/MemberView.tsx

```diff
@@ -53,35 +71,32 @@ export const MemberView: React.FC = () => {
   const member = memberships?.find((member) => member.handle === handle)
   const { url: avatarUrl, isLoadingAsset: avatarLoading } = useMemberAvatar(member)
 
-  const filtersBarLogic = useFiltersBar()
-  const {
-    filters: { setIsFiltersOpen, isFiltersOpen },
-    canClearFilters: { canClearAllFilters },
-  } = filtersBarLogic
-
   const toggleFilters = () => {
     setIsFiltersOpen((value) => !value)
   }
   const handleSorting = (value?: unknown) => {
     if (value) {
-      setSortVideosBy(value as VideoOrderByInput)
+      setSortVideosBy(value as OwnedNftOrderByInput)
     }
   }
   const handleSetCurrentTab = async (tab: number) => {
     setSearchParams({ 'tab': TABS[tab] }, { replace: true })
   }
 
-  const mappedTabs = TABS.map((tab) => ({ name: tab, pillText: tab === 'NFTs owned' ? '0' : undefined }))
+  const mappedTabs = TABS.map((tab) => ({
+    name: tab,
+    pillText: tab === 'NFTs owned' ? (nfts ? nfts.length : undefined) : undefined,
```

Comment:
`pillText: tab === 'NFTs owned' && nfts ? nfts.length : undefined,` save the nested operator?